src/libraries/System.Private.CoreLib/src/System/Enum.EnumInfo.cs

```diff
@@ -17,6 +18,17 @@ public EnumInfo(bool hasFlagsAttribute, ulong[] values, string[] names)
                 HasFlagsAttribute = hasFlagsAttribute;
                 Values = values;
                 Names = names;
+
+                // Store whether all of the values are sequential starting from zero.
+                ValuesAreSequentialFromZero = true;
+                for (int i = 0; i < values.Length; i++)
```

Comment:
Isn't there some vector operation that could do this too?

Reply:
There is, but 
* it is quite a bit amount of code
* the `EnumInfo` gets cached, so not called very often
* Vector128 for `ulong` takes 2 elements, so speedup won't be that big